sdk/maps/maps-search/review/maps-search.api.md

```diff
@@ -0,0 +1,756 @@
+    searchNearbyPointOfInterest(coordinates: LatLon, options?: SearchNearbyPointOfInterestOptions): Promise<SearchAddressResult>;
+    searchPointOfInterest(searchQuery: SearchQuery, options?: SearchPointOfInterestOptions): Promise<SearchAddressResult>;
+    searchPointOfInterestCategory(searchQuery: SearchQuery, options?: SearchPointOfInterestCategoryOptions): Promise<SearchAddressResult>;
+    searchStructuredAddress(structuredAddress: StructuredAddress, options?: SearchStructuredAddressOptions): Promise<SearchAddressResult>;
+}
+
+// @public
+export type MapsSearchClientOptions = CommonClientOptions & {
+    endpoint?: string;
+};
+
+// @public
+export type MatchType = string;
+
+// @public
+export interface OperatingHours {
+    readonly mode: string;
+    readonly timeRanges: OperatingHoursTimeRange[];
+}
+
+// @public
+export type OperatingHoursRange = string;
+
+// @public
+export interface OperatingHoursTime {
+    readonly date: string;
+    readonly hour: number;
+    readonly minute: number;
+}
+
+// @public
+export interface OperatingHoursTimeRange {
+    endTime: OperatingHoursTime;
+    startTime: OperatingHoursTime;
+}
+
+// @public
+export interface PointOfInterest {
+    readonly brands?: Brand[];
+    readonly categories?: string[];
+    readonly categorySet?: PointOfInterestCategorySet[];
+    readonly classifications?: Classification[];
+    readonly name: string;
+    operatingHours?: OperatingHours;
+    readonly phone?: string;
+    readonly url?: string;
+}
+
+// @public
+export interface PointOfInterestCategory {
+    readonly childIds: number[];
+    readonly id: number;
+    readonly name: string;
+    readonly synonyms: string[];
+}
+
+// @public
+export interface PointOfInterestCategorySet {
+    readonly id: number;
+}
+
+// @public
+export interface PointOfInterestCategoryTreeResult {
+    readonly categories?: PointOfInterestCategory[];
+}
+
+// @public
+export type Position = Position2D | Position3D;
+
+// @public
+export type Position2D = [longitude: number, latitude: number];
+
+// @public
+export type Position3D = [longitude: number, latitude: number, elevation: number];
+
+// @public
+export type QueryType = string;
+
+// @public
+export type ReverseSearchAddressBatchOptions = OperationOptions & BatchPollerOptions;
+
+// @public
+export interface ReverseSearchAddressOptions extends ReverseSearchBaseOptions {
+    allowFreeformNewline?: boolean;
+    entityType?: GeographicEntityType;
+    includeMatchType?: boolean;
+    includeRoadUse?: boolean;
+    includeSpeedLimit?: boolean;
+    roadUse?: RoadUseType[];
+    streetNumber?: string;
+}
+
+// @public
+export interface ReverseSearchAddressRequest {
+    // (undocumented)
+    coordinates: LatLon;
+    // (undocumented)
+    options?: ReverseSearchAddressOptions;
+}
+
+// @public (undocumented)
+export interface ReverseSearchAddressResult {
+    readonly numResults: number;
```

Comment:
`numberResult` sounds better?

Reply:
Here we try to say, "The number(size) of the `results` field." That's why we use the plural here. But not sure if this makes sense. I would like to know your thought on this. Thank you :)

Reply:
ah sorry for the typo, I mean `numberResults` vs `numResults`